hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -301,6 +301,18 @@ public void printReport() {
     messages.forEach(print);
   }
 
+  /**
+   * Print out reports with the given message.
+   */
+  public void print(String msg){
+    List<String> messages = new LinkedList<>();
+    messages.add(msg);
```

Comment:
Given we only have one String msg here, we can eliminate thew List allocation here by just simply call
print.accept(msg) in line 313.

Reply:
Thanks @xiaoyuyao. Fixed the print function.